isis/tests/PvlObjectTests.cpp

```diff
@@ -194,4 +195,37 @@ TEST(PvlObject, constructFromPvl) {
   EXPECT_EQ(p.findKeyword("numkey"), numkey);
 }
 
+TEST(PvlObject, PvlGroupEqualTest){
+  PvlGroup pvlTmplGrp("Point_ErrorMagnitude");
+  PvlKeyword pvlTmplKwrd("Point_ErrorMagnitude__Required", "true");
+  PvlKeyword pvlTmplKwrdOne("some_message", "true");
+  PvlKeyword pvlTmplKwrdTwo("foo", "true");
+  PvlKeyword pvlTmplKwrdThree("bar", "false");
+  PvlKeyword pvlTmplKwrdFour("fooie", "true");
+  pvlTmplGrp += pvlTmplKwrd;
+
+  PvlGroup copy = PvlGroup(pvlTmplGrp);
+
+  EXPECT_PRED_FORMAT2(AssertPvlGroupEqual, pvlTmplGrp, copy); // should pass
+
+
+  copy.addKeyword(pvlTmplKwrdOne);
+
+  EXPECT_FALSE(AssertPvlGroupEqual("Point_ErrorMagnitude", "Point_ErrorMagnitude", copy, pvlTmplGrp)); // should fail
```

Comment:
There should be a different `EXPECT_PRED_FORMAT{num}` where num is the number of arguments. So in this case, we should be able to run something like `EXPECT_PRED_FORMAT4(AssertPvlGroupEqual("Point_ErrorMagnitude"...`